### ISSUE TITLE
Placing the Navigation Component on a brand new project results in an error

### ISSUE THREAD

**Issue by LucaNerlich** [adobe/aem-core-wcm-components#1445]
### Expected Behaviour

The Navigation Component can be placed on either a template or a page.

### Actual Behaviour
The component throws an error and does not correctly populate its fields and exports them to the model.json.

#### Steps to Reproduce

1. Create a new AEM Project using the current archetype and cloud version.
2. Try to place the Core Navigation Component on either a page or a template

However, creating an experience fragment and placing the Navigation Component inside and then using this exp variation on a page works.

#### Platform and Version

Archetype v25

##### Archetype Properties
`
groupId=com.adobe
artifactId=spa-demo
version=1.0.0-SNAPSHOT
package=com.spa
appId=spa-demo
appTitle=SPA Demo
aemVersion=cloud
sdkVersion=latest
languageCountry=${languageCountry}
includeExamples=y
includeErrorHandler=n
frontendModule=react
singleCountry=y
includeDispatcherConfig=y
includeCommerce=n
includeForms=n
sdkFormsVersion=latest
commerceEndpoint=https://hostname.com/graphql
datalayer=y
amp=n
enableDynamicMedia=n
`

#### Logs taken while reproducing problem

```
11.03.2021 20:33:55.841 *ERROR* [[0:0:0:0:0:0:0:1] [1615491235832] GET /conf/spa-demo/settings/wcm/templates/spa-app-template/structure.html HTTP/1.1] org.apache.sling.scripting.sightly.render.ObjectModel Cannot access method brandSlug on object com.adobe.aem.spa.project.core.internal.impl.PageImpl@1e192cc
java.lang.reflect.InvocationTargetException: null
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

Caused by: java.lang.UnsupportedOperationException: null
	at com.adobe.cq.wcm.core.components.models.Page.getBrandSlug(Page.java:238) [com.adobe.cq.core.wcm.components.core:2.15.0]
```

In January two PRs regarding brandSlug have been worked on: https://github.com/adobe/aem-core-wcm-components/issues?q=brandSlug

**Comment by vladbailescu** [comment]
This should be fixed by #1435. @LucaNerlich , could you give it a try and cofirm?

**Comment by LucaNerlich** [comment]
My mac seems to have some issues, I cannot get a current instance running successfully, however Robert Botha also tested this and he confirmed, that it works now. Thanks for the fix :)
